src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.resx

```diff
@@ -4617,4 +4617,4 @@
   <data name="SqlRetryLogic_InvalidMinMaxPair" xml:space="preserve">
     <value>'{0}' is not less than '{1}'; '{2}' cannot be greater than '{3}'.</value>
   </data>
-</root>
+</root>
```

Comment:
NIT: same here can you add an extra line please.

Reply:
Why don't you have stylecop enabled for this?

Reply:
We have an editor config file at the root of our driver. I will look into see how that has been configured.

Reply:
This is a resource file so even if we add new lines manually, they'll get terminated when you edit something again. I'd rather keep it IDE friendly way.